examples/multisite_neuroimg_adapt/main.py

```diff
@@ -0,0 +1,97 @@
+"""
+Autism Detection: Domain Adaptation for Multi-Site Neuroimaging Data Analysis
+
+Reference:
+[1] Cameron Craddock, Yassine Benhajali, Carlton Chu, Francois Chouinard, Alan Evans, András Jakab, Budhachandra Singh Khundrakpam, John David Lewis, Qingyang Li, Michael Milham, Chaogan Yan, Pierre Bellec (2013). The Neuro Bureau Preprocessing Initiative: open sharing of preprocessed neuroimaging data and derivatives. In Neuroinformatics 2013, Stockholm, Sweden.
+
+[2] Abraham, A., Pedregosa, F., Eickenberg, M., Gervais, P., Mueller, A., Kossaifi, J., ... & Varoquaux, G. (2014). Machine learning for neuroimaging with scikit-learn. Frontiers in neuroinformatics, 14.
+
+[3] Zhou, S., Li, W., Cox, C.R., & Lu, H. (2020). Side Information Dependence as a Regularizer for Analyzing Human Brain Conditions across Cognitive Experiments. in AAAI 2020, New York, USA. https://ojs.aaai.org//index.php/AAAI/article/view/6179
+
+[4] Zhou, S. (2022). Interpretable Domain-Aware Learning for Neuroimage Classification (Doctoral dissertation, University of Sheffield). https://etheses.whiterose.ac.uk/31044/1/PhD_thesis_ShuoZhou_170272834.pdf
+"""
+import argparse
+import os
+
+import numpy as np
+import pandas as pd
+from config import get_cfg_defaults
+from nilearn.connectome import ConnectivityMeasure
+from nilearn.datasets import fetch_abide_pcp
+from sklearn.linear_model import RidgeClassifier
+
+import kale.utils.seed as seed
+from kale.evaluate import cross_validation
+from kale.pipeline.multi_domain_adapter import CoIRLS
+
+
+def arg_parse():
+    parser = argparse.ArgumentParser(
+        description="Autism Detection: Domain Adaptation for Multi-Site Neuroimaging Data Analysis"
+    )
+    parser.add_argument("--cfg", required=True, help="path to config file", type=str)
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = arg_parse()
+
+    # ---- Set up configs ----
+    cfg = get_cfg_defaults()
+    cfg.merge_from_file(args.cfg)
+    cfg.freeze()
+    seed.set_seed(cfg.SOLVER.SEED)
+
+    # ---- Fetch ABIDE fMRI timeseries ----
+    fetch_abide_pcp(
+        data_dir=cfg.DATASET.ROOT,
+        pipeline=cfg.DATASET.PIPELINE,
+        band_pass_filtering=True,
+        global_signal_regression=False,
+        derivatives=cfg.DATASET.ATLAS,
+        quality_checked=False,
+        SITE_ID=cfg.DATASET.SITE_IDS,
```

Comment:
I haven't found the parameter `SITE_ID` in the function `fetch_abide_pcp`. Please let me know if I miss it somehow.

Reply:
`SITE_ID` is a column name of the phenotype data